app/scripts/components/common/cookie-consent/index.tsx

```diff
@@ -73,33 +72,31 @@ export const CookieConsent = ({
 
     setCookie(cookieValue, closeConsent);
     onFormInteraction();
-    // }
-  }, [cookieConsentResponded, cookieConsentAnswer]);
+  }, [cookieConsentResponded, cookieConsentAnswer, closeConsent]);
```

Comment:
any reason `closeConset` was not a part of dependencies? (Also there are still dependencies missing - but I mainly wanted to check in with closeConsent since it is a dynamic value)

Reply:
The absence of `closeConsent` was a miss on my part. My original intent was to rely only on the actual cookie values to trigger the useEffect but it makes more sense to add that as a dependency since it influences the expiration of the actual cookie.